frontend/public/components/utils/details-page.tsx

```diff
@@ -63,12 +64,17 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({
     name: metadata.name,
     namespace: metadata.namespace,
   });
+  const { t } = useTranslation();
 
   return (
     <dl data-test-id="resource-summary" className="co-m-pane__details">
-      <DetailsItem label="Name" obj={resource} path={customPathName || 'metadata.name'} />
+      <DetailsItem
+        label={t('details-page~Name')}
```

Comment:
@rebeccaalpert FYI, this will conflict with the work you're doing on common components.